.github/workflows/build.yml.in

```diff
@@ -169,99 +169,7 @@ jobs:
           GITHUB_CONTEXT: ${{ toJson(github) }}
           WORKFLOW_URL: ${{ github.event.workflow_run.html_url }}
         run: |
-          # Extract branch name and check if it matches the pattern
-          branch_name=$(echo "$GITHUB_CONTEXT" | jq -r .ref | cut -d'/' -f3)
-          if [[ "$branch_name" =~ ^lean-pr-testing-([0-9]+)$ ]]; then
-            pr_number="${BASH_REMATCH[1]}"
-            current_time=$(date "+%Y-%m-%d %H:%M:%S")
-
-            echo "This is a 'lean-pr-testing-$pr_number' branch, so we need to adjust labels and write a comment."
-
-            if ${{ steps.lint.outcome == 'success' }}; then
```

Comment:
You'll need to move `${{ steps.lint.outcome == 'success' }}` into an environment variable (or a command line argument) if you want the separate script to have access to it

Reply:
Thanks. This also caught a `built` / `build` typo.